inference-engine/include/details/os/win_shared_object_loader.h

```diff
@@ -62,8 +124,11 @@ class SharedObjectLoader {
      */
     explicit SharedObjectLoader(LPCWSTR pluginName) {
         ExcludeCurrentDirectory();
+        auto oldDir = IncludePluginDirectoryW(pluginName);
```

Comment:
ExcludeCurrentDirectory calls SetDllDirectoryW(""). Could you debug and confirm IncludePluginDirectoryW returns "" (not nullptr) such that consequent SetDllDirectoryW will have the effect of ExcludeCurrentDirectory?